napi.h

```diff
@@ -850,18 +846,16 @@ namespace Napi {
     void Queue();
     void Cancel();
 
-    virtual void Execute() = 0;
-    virtual void WorkComplete();
-
     ObjectReference& Persistent();
 
   protected:
     explicit AsyncWorker(const Function& callback);
```

Comment:
I had been thinking that another object reference would be wasteful, but now I realize it could make sense to use the receiver object as the `_persistent` object reference, if one was supplied, instead of creating a new object. I'll update this PR.